include/utils.h

```diff
@@ -1,4 +1,7 @@
 #pragma once
 
 void InvalidateEntireInstructionCache (void);
+void CleanEntireDataCache (void);
```

Comment:
Missing declarations from utils.s. I think there is no InvalidateEntireDataCache, however I did not delete since this does not seem to be used.